checkov/terraform/checks/resource/azure/MySQLPublicAccessDisabled.py

```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class MySQLPublicAccessDisabled(BaseResourceCheck):
```

Comment:
@JamesWoolfenden I think it be inherited from baseresourcevaluecheck

Reply:
made it so